src/homeworkNumber6/countingRhyme.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct Node {
+    int id;
+    struct Node* next;
+} Node;
+
+int main(void)
+{
+    int n, m;
+    printf("Введите n (число воинов) и m (шаг убийства): ");
+    if (scanf("%d %d", &n, &m) != 2 | n <= 0 | m <= 0) {
```

Comment:
```suggestion
    if (scanf("%d %d", &n, &m) != 2 || n <= 0 || m <= 0) {
```
Не путайте, пожалуйста, побитовые и логические операции.